### PR TITLE
feat: add borderRadius function to theme and adjust barscale

### PR DESCRIPTION
**Pull request by mjmonline**

### CODE REVIEW

File: src/theme/theme.ts [hunk 1]
```diff
@@ -92,6 +92,8 @@ export const createTheme = (config: ThemeConfig = {}): Theme => {
   const RADIUS_LARGE: BORDER_RADIUS = 20;
   const RADIUS_100: BORDER_RADIUS = 100;
 
+  const borderRadius: Theme['borderRadius'] = (x) =>
```
Comment (emilwestin) [inline src/theme/theme.ts:95]: Should `x` be typed to be a `BORDER_RADIUS` so we "lock" what type of radius you are allowed to input?

Comment (emilwestin) [reply src/theme/theme.ts:95]: Or is the purpose of this the be able to override and do sent in whatever you like?

Comment (emilwestin) [reply src/theme/theme.ts:95]: Nvm.. see that you typed it below..! Disregard this :D

Comment (mjmonline) [reply src/theme/theme.ts:95]: it is already when we type the whole function to `Theme['borderRadius']`

**Review by emilwestin — APPROVED** [review]

**Pull request merged**